src/main/java/net/minecraftforge/network/VanillaConnectionNetworkFilter.java

```diff
@@ -84,7 +85,7 @@ public static void injectIfNecessary(NetworkManager manager)
     @Nonnull
     private static SEntityPropertiesPacket filterEntityProperties(SEntityPropertiesPacket msg)
     {
-        SEntityPropertiesPacket newPacket = new SEntityPropertiesPacket();
+        SEntityPropertiesPacket newPacket = new SEntityPropertiesPacket(msg.getEntityId(), Collections.emptyList());
```

Comment:
Won't using `Collections.emptyList` throw an `UnsupportedOperationException` when adding the snapshots further down the line? Shouldn't we use a new ArrayList?

Reply:
`SEntityPropertiesPacket` always initializes its `snapshot` field as `Lists.newArrayList`. This constructor parameter is only used in a for loop to fill that snapshots list. However as we already have a list of snapshots here (and not a list of `ModifiableAttributeInstance`, I pass in an empty list so the constructor does not add anything to the snapshots list - we will do that later manually.

Reply:
You are right, for some reason I misread the code there.